new-log-viewer/src/utils/actions.ts

```diff
@@ -0,0 +1,65 @@
+import * as monaco from "monaco-editor/esm/vs/editor/editor.api.js";
+
+import {Nullable} from "../typings/common";
+
+
+enum ACTION {
+    FIRST_PAGE = "firstPage",
+    PREV_PAGE = "prevPage",
+    NEXT_PAGE = "nextPage",
+    LAST_PAGE = "lastPage",
+    PAGE_TOP = "pageTop",
+    PAGE_BOTTOM = "pageBottom"
+}
+
+type ActionType = {
+    action: Nullable<ACTION>,
+    label: string,
+    keybindings: monaco.KeyCode[],
```

Comment:
`keyBindings`? I know Monaco calls it `keybindings`, but perhaps we should stick to our own guidelines unless there's another reason to stick with lowercase.

Reply:
I thought that was a word since PyCharm did not warn about any spelling errors, lol.

Looking it up online, it seems we should either call it `keybinds` or `key bindings`: https://en.wiktionary.org/wiki/keybind As I am typing this, Microsoft Edge already warns about the `keybinds` spelling.

Reply:
Let's stick with `keyBindings` then.